src/commands/MentionCommand.ts

```diff
@@ -32,6 +32,20 @@ export default class MentionCommand extends Command {
 			);
 		}
 
+		if ( !BotConfig.quotedTicketsCauseEmbed ) {
+			const lineArray = messageText.split( '\n' );
+			const removedLines: Array<string> = [];
+			for ( const line of lineArray ) {
+				if ( line.startsWith( '> ' ) ) {
+					removedLines.push( line );
+				}
+			}
+			for ( const remove of removedLines ) {
+				lineArray.splice( lineArray.indexOf( remove ), 1 );
+			}
+			messageText = lineArray.join( '\n' );
```

Comment:
This can be simplified a lot.
```suggestion
			messageText = messageText
				.split( '\n' )
				.filter( line => !line.startsWith( '> ' ) )
				.join( '\n' );
```